tests/unit/OptionsListUtilsTest.js

```diff
@@ -221,26 +221,27 @@ describe('OptionsListUtils', () => {
         results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, 'peterparker@expensify.com');
 
         // Then one option will be returned and it will be the correct option
-        expect(results.personalDetails.length).toBe(1);
-        expect(results.personalDetails[0].text).toBe('Spider-Man');
+        expect(results.recentReports.length).toBe(1);
+        expect(results.recentReports[0].text).toBe('Spider-Man');
 
         // When we provide a search value that matches a partial display name or email
         results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, 'man');
 
         // Then several options will be returned and they will be each have the search string in their email or name
         // even though the currently logged in user matches they should not show.
-        expect(results.personalDetails.length).toBe(3);
-        expect(results.personalDetails[0].text).toBe('Spider-Man');
-        expect(results.personalDetails[1].text).toBe('Invisible Woman');
-        expect(results.personalDetails[2].login).toBe('natasharomanoff@expensify.com');
+        expect(results.personalDetails.length).toBe(1);
+        expect(results.recentReports.length).toBe(2);
+        expect(results.personalDetails[0].login).toBe('natasharomanoff@expensify.com');
+        expect(results.recentReports[0].text).toBe('Invisible Woman');
+        expect(results.recentReports[1].text).toBe('Spider-Man');
 
         // Test for Concierge's existence in chat options
         results = OptionsListUtils.getNewChatOptions(REPORTS_WITH_CONCIERGE, PERSONAL_DETAILS_WITH_CONCIERGE);
 
-        // Concierge is included in the results by default and all the personalDetails should be returned
-        // minus the currently logged in user
-        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS_WITH_CONCIERGE) - 1);
-        expect(results.personalDetails).toEqual(
+        // Concierge is included in the results by default. We should expect all the personalDetails to show
+        // (minus the 5 that are already showing and the currently logged in user)
+        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS_WITH_CONCIERGE) - 1 - 5);
+        expect(results.recentReports).toEqual(
```

Comment:
can't we add the test for `recentReports`, without deleting the personalDetails test?

Reply:
`CONCIERGE` doesn't exist in `personalDetails`, it exists in `recentReports`.
So, I think there's no point to check what the `personalDetails` to be equal yet I check its length

Reply:
Ah, I see. Fair enough